src/libcore/ptr/mod.rs

```diff
@@ -119,10 +119,13 @@ mod mut_ptr;
 ///
 /// Behavior is undefined if any of the following conditions are violated:
 ///
-/// * `to_drop` must be [valid] for reads.
+/// * `to_drop` must be [valid] for both reads and writes.
 ///
 /// * `to_drop` must be properly aligned.
 ///
+/// * The value `to_drop` points to must be valid for destructing, which may mean it must uphold
```

Comment:
Maybe "dropping" would be a better, more Rust-specific term?

Reply:
Done.